test/integration/commands/validate_data/test_reformat_validaton_result.py

```diff
@@ -26,31 +28,163 @@ def result_cardinality_violation() -> Graph:
     return g
 
 
-def test_reformat_validation_graph(result_cardinality_violation: Graph, data_cardinality_violation: Graph) -> None:
+@pytest.fixture
+def data_value_type_violation() -> Graph:
+    g = Graph()
+    g.parse("testdata/validate-data/validation_results/value_type_violation_data.ttl")
+    return g
+
+
+@pytest.fixture
+def result_value_type_violation() -> Graph:
+    g = Graph()
+    g.parse("testdata/validate-data/validation_results/value_type_violation_result.ttl")
+    return g
+
+
+@pytest.fixture
+def data_content_violation() -> Graph:
+    g = Graph()
+    g.parse("testdata/validate-data/validation_results/content_violation_data.ttl")
+    return g
+
+
+@pytest.fixture
+def result_content_violation() -> Graph:
+    g = Graph()
+    g.parse("testdata/validate-data/validation_results/content_violation_result.ttl")
+    return g
+
+
+@pytest.fixture
+def data_every_constraint_once() -> Graph:
+    g = Graph()
+    g.parse("testdata/validate-data/validation_results/every_combination_once_data.ttl")
+    return g
+
+
+@pytest.fixture
+def result_card_every_constraint_once() -> Graph:
+    g = Graph()
+    g.parse("testdata/validate-data/validation_results/every_combination_once_result_cardinality.ttl")
+    return g
+
+
+@pytest.fixture
+def result_content_every_constraint_once() -> Graph:
+    g = Graph()
+    g.parse("testdata/validate-data/validation_results/every_constraint_once_result_content.ttl")
```

Comment:
*idea* Align fixture function names with file names.

The function names and the file names do not seem to follow a strict system for their naming. IMHO it would be beneficial to have the function named the same way the file is named. This would make it easy when reading the test to open the test file without having to navigate to the fixture method and read it first.